spec/models/workshop_spec.rb

```diff
@@ -3,21 +3,27 @@
 RSpec.describe Workshop, type: :model do
   describe "validations" do
     it "is invalid without a title" do
-      workshop = Workshop.new(title: "")
+      workshop = FactoryGirl.build(:workshop, title: "")
```

Comment:
if you added Factory to your spec helper, you could (just) write  `workshop = build(:workshop, title: "")`